Tweak/Tweak.x

```diff
@@ -11,7 +11,8 @@ UIColor *currentBundleColor = nil;
 %hook _UIContextMenuActionsListView
 
 - (void)willMoveToWindow:(UIWindow *)newWindow {
-	[[([self.subviews objectAtIndex:0]).subviews objectAtIndex:0] setBackgroundColor:currentBundleColor];
+	if ([self.subviews count] && [self.subviews objectAtIndex:0] && [([self.subviews objectAtIndex:0]).subviews count])
+		[[([self.subviews objectAtIndex:0]).subviews objectAtIndex:0] setBackgroundColor:currentBundleColor];
```

Comment:
I'd check if it responds to setBackgroundColor selector too, or being a kind of class, once it's used